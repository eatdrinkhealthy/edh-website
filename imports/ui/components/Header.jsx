import React from "react";

const Header = () => (
  <header className="app-header">
    <div className="row">
      <div className="column">
        <div className="app-header__top">
          <a href="/">
            <img src="images/logo.png" alt="Company Logo" />
          </a>

          <nav className="app-header-nav">
            <ul className="menu align-center">
              <li className="is-active">
                <a href="/">HOME</a>
              </li>
              <li>
                <a href="#contact">CONTACT</a>
              </li>
              <li>
                <a href="/privacy-policy">PRIVACY</a>
              </li>
            </ul>
          </nav>

          <div className="app-header-social text-center medium-text-right">
            <a href="mailto:info@eatdrinkhealthy.co">
              <span className="fa-stack fa-lg">
                <i className="fa fa-circle fa-stack-2x" />
                <i className="fa fa-envelope fa-stack-1x fa-inverse" />
              </span>
            </a>
            <a
              href="https://www.linkedin.com/company/16173480/"
              target="_blank"
              rel="noopener noreferrer"
            >
              <span className="fa-stack fa-lg">
                <i className="fa fa-circle fa-stack-2x" />
                <i className="fa fa-linkedin fa-stack-1x fa-inverse" />
              </span>
            </a>
          </div>
        </div>
      </div>
    </div>

    <section className="app-header__banner">
      <div className="app-header__banner-content-wrapper">
        <div className="row">
          <div className="column medium-8 large-7">
            <div className="app-header__banner-headline">
              A platform to find, share, and discuss healthy places to shop and eat.
            </div>
            <div className="app-header__banner-buttons">
              <a
                href="https://app.eatdrinkhealthy.co/"
                target="_blank"
                rel="noopener noreferrer"
                className="button button--app-store"
              >
                <i className="fa fa-apple" />
                <span>
                  Download on<br />the <strong>App Store</strong>
                </span>
              </a>
              <a
                href="https://itunes.apple.com/app/id1116167404"
                target="_blank"
                rel="noopener noreferrer"
                className="button button--wep-app"
              >
                <span>Web App</span>
              </a>
            </div>
          </div>
        </div>
      </div>
    </section>
  </header>
);

export default Header;
