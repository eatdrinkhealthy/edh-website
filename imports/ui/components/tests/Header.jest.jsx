/* eslint-env jest */
/* eslint-disable func-names, prefer-arrow-callback, no-unused-expressions */
import React from "react";
import renderer from "react-test-renderer";
import Header from "../Header";

describe("<Header />", () => {
  it("should match render snapshot", () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
