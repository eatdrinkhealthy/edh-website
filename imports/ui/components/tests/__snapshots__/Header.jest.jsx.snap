// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should match render snapshot 1`] = `
<div>
  <div
    className="row"
  >
    <div
      className="column"
    >
      <div
        className="app-header__top"
      >
        <a
          href="/"
        >
          <img
            alt="Company Logo"
            src="images/logo.png"
          />
        </a>
        <nav
          className="app-header-nav"
        >
          <ul
            className="menu align-center"
          >
            <li>
              <a
                aria-current="true"
                className="active"
                href="/"
                onClick={[Function]}
                style={
                  Object {
                    "color": "#f28020",
                  }
                }
              >
                HOME
              </a>
            </li>
            <li>
              <a
                aria-current={false}
                className={undefined}
                href="#contactForm"
                onClick={[Function]}
                style={undefined}
              >
                CONTACT
              </a>
            </li>
            <li>
              <a
                aria-current={false}
                className={undefined}
                href="/privacy-policy"
                onClick={[Function]}
                style={undefined}
              >
                PRIVACY
              </a>
            </li>
          </ul>
        </nav>
        <div
          className="app-header-social text-center medium-text-right"
        >
          <a
            href="mailto:info@eatdrinkhealthy.co"
          >
            <span
              className="fa-stack fa-lg"
            >
              <i
                className="fa fa-circle fa-stack-2x"
              />
              <i
                className="fa fa-envelope fa-stack-1x fa-inverse"
              />
            </span>
          </a>
          <a
            href="https://www.linkedin.com/company/16173480/"
            rel="noopener noreferrer"
            target="_blank"
          >
            <span
              className="fa-stack fa-lg"
            >
              <i
                className="fa fa-circle fa-stack-2x"
              />
              <i
                className="fa fa-linkedin fa-stack-1x fa-inverse"
              />
            </span>
          </a>
        </div>
      </div>
    </div>
  </div>
  <header
    className="app-header"
  >
    <section
      className="app-header__banner"
    >
      <div
        className="app-header__banner-content-wrapper"
      >
        <div
          className="row"
        >
          <div
            className="column medium-8 large-7"
          >
            <div
              className="app-header__banner-headline"
            >
              A platform to find, share, and discuss healthy places to shop and eat.
            </div>
            <div
              className="app-header__banner-buttons"
            >
              <a
                className="button button--app-store"
                href="https://itunes.apple.com/app/id1116167404"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fa fa-apple"
                />
                <span>
                  Download on
                  <br />
                  the 
                  <strong>
                    App Store
                  </strong>
                </span>
              </a>
              <a
                className="button button--wep-app"
                href="https://app.eatdrinkhealthy.co/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <span>
                  Web App
                </span>
              </a>
            </div>
          </div>
        </div>
      </div>
    </section>
  </header>
</div>
`;
